ports/mimxrt/boards/MIMXRT1064_EVK/mpconfigboard.mk

```diff
@@ -0,0 +1,9 @@
+MCU_SERIES = MIMXRT1064
+MCU_VARIANT = MIMXRT1064DVL6A
+
+JLINK_PATH = /media/RT1064-EVK/
```

Comment:
other boards have this as `JLINK_PATH ?= ...`

Reply:
Will update